### ISSUE TITLE
Error: $ operator is invalid for atomic vectors

### ISSUE THREAD

**Issue by praneetheranti** [karthik/rdrop2#162]
library(tm)
library(skmeans)
options(header=FALSE ,stringsAsFactors = FALSE, fileEncoding= "Latin1")
data <- read.csv("E:\\python\\Incident\\Output\\Count.csv")

corpus1<-Corpus(VectorSource(data$Description))

inspect(corpus1)

cleanset <- tm_map(corpus1 , removeWords , stopwords("english")  )
cleanset <- tm_map(cleanset , stripWhitespace  )

dtm <- DocumentTermMatrix(cleanset)

dtm_tfxidf <- weightTfIdf(dtm)

m <- as.matrix(dtm_tfxidf)
rownames(m) <- 1:nrow(m)

norm_eucl <- function(m)
  m/apply(m,1,function(x) sum(x^2)^.5)

m_norm <- norm_eucl(m)
results <- kmeans(m_norm ,20, 40)
clusters <- 1:20

for(i in clusters) {
 
  cat("Cluster ",i,":", findMostFreqTerms(dtm_tfxidf[(results$cluster)==i],2,"\n\n" ))
}

Basically, I am trying to do cluster analysis using kmeans
When i am running above code. i am encountering with error "Error: $ operator is invalid for atomic vectors"

Can some one help here ,how to fix this ? $ symbol i am using on last but one line i.e. (results$cluster).
